### PR TITLE
locale(de): update german translation

### PR DESCRIPTION
**Pull request by meteyou**
## Description

Update german locale file.

## Related Tickets & Documents

none

## Mobile & Desktop Screenshots/Recordings

none

## [optional] Are there any post-deployment tasks we need to perform?

none

### CODE REVIEW

File: src/locales/de.json [hunk 1]
```diff
@@ -316,22 +347,35 @@
         "Wireframe": "Drahtgitter"
     },
     "History": {
+        "AddANote": "Eine Nitiz hinzufügen",
```
Comment (dw-0) [inline src/locales/de.json:350]:
```suggestion
        "AddANote": "Eine Notiz hinzufügen",
```

**Review by dw-0 — CHANGES_REQUESTED** [review]

File: src/locales/de.json (hunk 1 above)
Comment (mryel00) [inline src/locales/de.json:350]: Typo. Correct would be `Eine Notiz hinzufügen`

File: src/locales/de.json [hunk 2]
```diff
@@ -28,14 +31,21 @@
             "NextReboot": "nächsten Reboot",
             "NoNotification": "Keine Benachrichtigung vorhanden",
             "Notifications": "Benachrichtigungen",
-            "Remind": "Errinnere:"
+            "OneDayShort": "1T",
+            "OneHourShort": "1Std",
+            "OneWeekShort": "1W",
+            "Remind": "Errinnere:",
```
Comment (mryel00) [inline src/locales/de.json:37]: Typo. Correct would be `Erinnere`

Comment (dw-0) [inline src/locales/de.json:37]:
```suggestion
            "Remind": "Erinnere:",
```

**Review by dw-0 — CHANGES_REQUESTED** [review]

File: src/locales/de.json [hunk 3]
```diff
@@ -146,6 +156,26 @@
...
         "SetupConsole": "Einstellungen"
     },
+    "CoolDownDialog": {
+        "AreYouSure": "Bist du sicher?",
+        "CoolDown": "Abkühlen",
+        "No": "Nein",
+        "Yes": "Ja"
+    },
+    "DevicesDialog": {
+        "CanBusInfo": "Nur nicht zugewiesene Knoten können erkannt werden. Es wird empfohlen, nur ein nicht zugewiesenes Gerät an den Can-Bus anzuschließen, um Kommunikationsprobleme zu vermeiden. Weitere Informationen finden Sie unter:",
```
Comment (mryel00) [inline src/locales/de.json:166]: Why formal language here? Above at `AreYouSure`, you used informal language.

**Review by mryel00 — CHANGES_REQUESTED** [review]

**Review by dw-0 — APPROVED** [review]

**Review by mryel00 — APPROVED** [review]

**Pull request merged**